gradle.properties

```diff
@@ -19,3 +21,9 @@ org.gradle.jvmargs=-Xmx1536M
 
 android.useAndroidX=true
 android.enableJetifier=false
+
+android.defaults.buildfeatures.buildconfig=false
+android.defaults.buildfeatures.aidl=false
+android.defaults.buildfeatures.renderscript=false
+android.defaults.buildfeatures.resvalues=false
+android.defaults.buildfeatures.shaders=false
```

Comment:
@JakeWharton I've been wondering how to do this for a while but never looked to deeply into it. Is there documentation for these properties somewhere?

Reply:
https://android.googlesource.com/platform/tools/base/+/mirror-goog-studio-master-dev/build-system/gradle-core/src/main/java/com/android/build/gradle/options/BooleanOption.kt

Reply:
Yep that's it. Sorry your initial comment got lost in the flurry of activity.